internal/common_flags/common_flags.go

```diff
@@ -0,0 +1,28 @@
+package common_flags
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	KafkaEndpointsFlagName              = "kafka-endpoints"
+	TeamsWebhookNotificationUrlFlagName = "teams-webhook-notification-url"
+)
+
+func ParseKafkaEndpointsFlags(cmd *cobra.Command) ([]string, error) {
+	kafkaEndpoints, kafkaEndpointsError := cmd.Flags().GetStringSlice(KafkaEndpointsFlagName)
+	if kafkaEndpointsError != nil {
+		return []string{}, fmt.Errorf("failed to get kafka-endpoints flag, cause: `%w`", kafkaEndpointsError)
+	}
+	return kafkaEndpoints, nil
+}
+
+func ParseTeamsWebhookNotificationUrlFlags(cmd *cobra.Command) (string, error) {
+	teamsWebhookNotificationUrl, teamsWebhookNotificationUrlError := cmd.Flags().GetString(TeamsWebhookNotificationUrlFlagName)
+	if teamsWebhookNotificationUrlError != nil {
+		return "", fmt.Errorf("failed to get teams-webhook-notification-url flag, cause: `%w`", teamsWebhookNotificationUrlError)
+	}
+	return teamsWebhookNotificationUrl, nil
+}
```

Comment:
```suggestion
var (
	KafkaEndpoints              []string
	TeamsWebhookNotificationUrl string
)
```

Together with the suggested change to `root.go` this will simplify code.
When you want to use the flag value, it is as simple as
```golang
fmt.Println(common_flags.KafkaEndpoints, common_flags.TeamsWebhookNotificationUrl)
```

Reply:
Thanks for the suggestions, since last time I looked at the code I think I changed my mind and agree with that this is much shorter and less error prone.